packages/editor/src/client/atoms/poll/pollAnswers.tsx

```diff
@@ -0,0 +1,179 @@
+import {ApolloError} from '@apollo/client'
+import PlusIcon from '@rsuite/icons/legacy/Plus'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import React, {useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Col, Form, IconButton, Message, Modal, Row, toaster} from 'rsuite'
+
+import {
+  FullPoll,
+  PollAnswer,
+  PollAnswerWithVoteCount,
+  useCreatePollAnswerMutation,
+  useDeletePollAnswerMutation
+} from '../../api'
+
+interface PollAnswersProps {
+  poll?: FullPoll
+  onPollUpdated(poll: FullPoll): void
+  onAnswerAdded(answer: PollAnswer): Promise<void>
+  onAnswerDeleted(): Promise<void>
+}
+
+export function PollAnswers({
+  poll,
+  onPollUpdated,
+  onAnswerAdded,
+  onAnswerDeleted
+}: PollAnswersProps) {
+  const {t} = useTranslation()
+  const [modalOpen, setModalOpen] = useState<boolean>(false)
+  const [answerToDelete, setAnswerToDelete] = useState<PollAnswerWithVoteCount | undefined>(
+    undefined
+  )
+  const [newAnswer, setNewAnswer] = useState<string>('')
+  const [createAnswerMutation, {loading}] = useCreatePollAnswerMutation()
+  const [deleteAnswerMutation] = useDeletePollAnswerMutation()
+
+  const onErrorToast = (error: ApolloError) => {
+    toaster.push(
+      <Message type="error" showIcon closable duration={3000}>
+        {error.message}
+      </Message>
+    )
+  }
+
+  /**
+   * FUNCTIONS
+   */
+  async function createAnswer() {
+    if (!poll) {
+      return
+    }
+    if (newAnswer === '') {
+      toaster.push(
+        <Message type="error" showIcon closable duration={3000}>
+          {t('pollAnswer.answerMissing')}
+        </Message>
+      )
+      return
+    }
+    const answer = await createAnswerMutation({
+      variables: {
+        pollId: poll.id,
+        answer: newAnswer
+      }
+    })
+    const savedAnswer = answer?.data?.createPollAnswer
+    if (savedAnswer) {
+      await onAnswerAdded(savedAnswer)
+    }
+    setNewAnswer('')
+  }
+
+  async function deleteAnswer(): Promise<void> {
+    setModalOpen(false)
+    if (!answerToDelete) {
+      return
+    }
+    await deleteAnswerMutation({
+      variables: {
+        deletePollAnswerId: answerToDelete.id
+      },
+      onError: onErrorToast
+    })
+    await onAnswerDeleted()
+  }
+
+  async function updateAnswer(updatedAnswer: PollAnswerWithVoteCount) {
+    if (!poll) {
+      return
+    }
+    const updatedAnswers = [...(poll.answers || [])]
```

Comment:
```suggestion
    const updatedAnswers = poll.answers ? [...poll.answers] : []
```

A bit more readable for me